src/components/Button/DislikeBox.tsx

```diff
@@ -28,12 +28,12 @@ export default React.memo(function DislikeBox(props: DislikeBoxProps) {
     >
       <Icon
         name="thumbs down outline"
-        className={active ? "button-box__icon" : "button-box__icon-hovered"}
+        className={!active ? "button-box__icon" : "button-box__icon-hovered"}
```

Comment:
Idem to `FavoriteBox`

```suggestion
        className={TokinList.join([
          "button-box__icon",
          !!active && "button-box__icon--active"
        ])}
```